pkgs/tools/misc/iotools/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, lib, fetchFromGitHub
+, gnumake, gcc10
+}:
+
+stdenv.mkDerivation rec {
+  pname = "iotools";
+  version = "1.6";
+
+  src = fetchFromGitHub {
+    owner = "adurbin";
+    repo = pname;
+    rev = "18949fdc4dedb1da3f51ee83a582b112fb9f2c71";
+    sha256 = "0vymnah44d5bzsjhfmxkcrlrikkp0db22k7a1s8bknz7glk9fldn";
+  };
+
+  nativeBuildInputs = [ gnumake gcc10 ];
+
```

Comment:
```suggestion

```

Sorry that I did not notice this earlier but they should be included in stdenv already.

Reply:
Ah okay. I removed it.